src/intl/en/learn-quizzes.json

```
@@ -324,5 +324,75 @@
+  "j002-prompt": "What happens if a validator goes offline?",
+  "j002-a-label": "No affect on rewards",
+  "j002-a-explanation": "Penalties are incurred when a validator is unavailable to attest to the state of the chain for any given epoch. The size of these penalties is approximately equal to 75% of what the reward for a proper attestation would have been. Rewards resume when validator goes back online, and NO slashing occurs.",
+  "j002-b-label": "Inactivity penalties are incurred only while unavailable",
+  "j002-b-explanation": "While unavailable, a validator will incur small inactivity penalties, approximately equal to 75% of what the reward would have been for a proper attestation. In rare/extreme cases where the network is not finalizing (i.e. over 1/3 of the network is also offline), these penalties are significantly greater. Rewards resume when validator goes back online, and no slashing occurs.",
+  "j002-c-label": "Immediate slashing and removal from the network",
+  "j002-c-explanation": "This is a common misconception, but going offline does NOT result in slashing! Slashing is a specific type of penalty for more serious offense, with larger penalties and also results in removal from the validator set.",
+  "j002-d-label": "One week delay before slashing and ejection",
+  "j002-d-explanation": "Being offline does NOT result in slashing, even after extended period of time. A validator could theoretically be offline for years without being slashed, though inactivity penalties would mount if the validator does not exit.",
+  "j003-prompt": "What is the max effective balance of a validator?",
+  "j003-a-label": "16",
+  "j003-a-explanation": "Validators who drop to an effective balance of 16 ETH are automatically exited from the Beacon Chain.",
+  "j003-b-label": "32",
+  "j003-b-explanation": "32 ETH is both the minimum ETH required to activate a new validator, and also the maximum 'effective balance' (vote weight) for that validator. Rewards above 32 can be accrued, but this balance does not contribute to the weight of that validators vote on the network and rewards are not increased.",
+  "j003-c-label": "Variable depending on the operator",
+  "j003-c-explanation": "The rules of consensus apply to every validator account equally and are not dependent upon the individual operating the node. The max effective balance of all validators is 32 ETH.",
+  "j003-d-label": "No limit",
+  "j003-d-explanation": "Each validator account is limited to an effective balance of 32 ETH, limiting the overall power of any single validator on the network. This also limits how much ETH can be staked or un-staked in a given time period, as validator activations and exits are processed through a rate-limited queue.",
+  "j004-prompt": "Which is NOT a reward received as a validator?",
```

Let's not arbitrarily capitalise words 😄

```suggestion
  "j004-prompt": "Which is not a reward received as a validator?",
```

---

It's not arbitrary =) it's a key word for negation... this technique is common on quizzes/tests to make sure people don't miss that key word

---

Although to your point, I didn't use this previously... I think it's a nice technique to user honestly, but not a hill I'm dying on—happy to lowercase.

---

I think the capitalized negation is pretty common for test questions. Since we can't bold, I'm not against this.